### PR TITLE
[v2.9] k8s October 2024 patches

### PR DESCRIPTION
**Pull request by chiukapoor**
### Updates
- Add `v1.28.15`, `v1.29.10`, `v1.30.6`
- Bump rke-tools to `v0.1.104`
- Bump nginx to `v1.11.3`
- Add nginx template for `v1.11.3`

### Issues
- https://github.com/rancher/rancher/issues/47536
- https://github.com/rancher/rancher/issues/47709

### CODE REVIEW

File: pkg/rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -11058,6 +11142,46 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.23.0",
 		},
+		// Enabled in Rancher v2.9.3
+		"v1.30.6-rancher1-1": {
```
Comment (kinarashah) [inline pkg/rke/k8s_rke_system_images.go:11146]: we'd need to change default in versions file to v1.30.6 too

File: pkg/rke/k8s_rke_system_images.go [hunk 2]
```diff
@@ -10730,6 +10730,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.8.9
```
Comment (kinarashah) [inline pkg/rke/k8s_rke_system_images.go:10733]: nit: Enabled in Rancher v2.9.3

Comment (kinarashah) [reply pkg/rke/k8s_rke_system_images.go:10733]: Oh nevermind, I see that we retain info of the min rancher server version it was introduced with

File: pkg/rke/k8s_rke_system_images.go (hunk 1 above)
Comment (chiukapoor) [reply pkg/rke/k8s_rke_system_images.go:11146]: Yes, that's what was causing the RKE PR to fail. Updated

**Review by kinarashah — APPROVED** [review]

**Comment by kinarashah** [comment]
Changes approved, verify RKE CI passed before merging https://github.com/rancher/rke/pull/3725

**Comment by kinarashah** [comment]
RKE CI passed.

**Pull request merged**